sdk/core/core-rest-pipeline/review/core-rest-pipeline.api.md

```diff
@@ -38,6 +40,36 @@ export interface BearerTokenAuthenticationPolicyOptions {
     scopes: string | string[];
 }
 
+// @public
+export function bearerTokenChallengeAuthenticationPolicy(options: BearerTokenChallengeAuthenticationPolicyOptions): PipelinePolicy;
+
+// @public
+export const bearerTokenChallengeAuthenticationPolicyName = "bearerTokenChallengeAuthenticationPolicy";
+
+// @public
+export interface BearerTokenChallengeAuthenticationPolicyOptions {
+    challengeCallbacks?: ChallengeCallbacks;
+    credential: TokenCredential;
+    scopes: string | string[];
```

Comment:
can we make this simply `string[]` instead of supporting both?

Reply:
Done